pkg/skaffold/runner/v1/new.go

```diff
@@ -92,7 +93,11 @@ func NewForConfig(runCtx *runcontext.RunContext) (*SkaffoldRunner, error) {
 
 	var podSelectors kubernetes.ImageListMux
 	var deployer deploy.Deployer
-	deployer, podSelectors, err = getDeployer(runCtx, labeller.Labels())
+	provider := deploy.ComponentProvider{
+		Logger: log.NewLogProvider(runCtx, kubectlCLI),
```

Comment:
Can we pass `out` here and remove the `out` from `Start` signature from Logger to keep `Start` and `Stop` same for `Logger`, `PortForwarder`, `DebugContainerManager`?

Reply:
unfortunately we can't, I wanted to do this too. we don't have access to the output stream at the time we call `NewForConfig`